src/uwtools/drivers/driver.py

```diff
@@ -66,27 +70,24 @@ def __repr__(self) -> str:
             h, r = divmod(self._leadtime.total_seconds(), 3600)
             m, s = divmod(r, 60)
             leadtime = "%02d:%02d:%02d" % (h, m, s)
-        return " ".join(
-            filter(None, [str(self), cycle, leadtime, "in", self._driver_config[STR.rundir]])
-        )
+        return " ".join(filter(None, [str(self), cycle, leadtime, "in", self.config[STR.rundir]]))
 
     def __str__(self) -> str:
         return self._driver_name
 
     @property
     def config(self) -> dict:
         """
-        Return a copy of the driver config.
+        A copy of the driver config.
         """
-        return deepcopy(self._driver_config)
+        return deepcopy(self._config)
```

Comment:
This property method (and `config_full()`, just below) are simpler and more consistent w.r.t. naming now.